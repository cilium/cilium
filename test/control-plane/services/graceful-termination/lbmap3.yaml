AffinityMatch: {}
BackendByID: {}
DummyMaglevTable: {}
ServiceByID:
  "1":
    Backends: []
    Frontend:
      ID: 1
      IP: 10.96.221.149
      Port: 8081
      Protocol: NONE
      Scope: 0
    HealthCheckNodePort: 0
    L7LBFrontendPorts: null
    L7LBProxyPort: 0
    LoadBalancerSourceRanges: null
    Name: ""
    Namespace: ""
    NatPolicy: ""
    SessionAffinity: false
    SessionAffinityTimeoutSec: 0
    TrafficPolicy: ""
    Type: ClusterIP
SourceRanges: {}
SvcActiveBackendsCount:
  "1": 0
