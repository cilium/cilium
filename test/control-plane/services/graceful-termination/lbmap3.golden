 - Services ----------------------------------------
| ID |      Type |           Frontend | Backend IDs |
|----+-----------+--------------------+-------------
|  0 | ClusterIP | 10.96.221.149:8081 |             |
 ---------------------------------------------------

 - Backends -----------------------
| ID | L3n4Addr | State | Restored |
|----+----------+-------+----------
 ----------------------------------

